slurm_pipe.py

```diff
@@ -206,9 +203,10 @@
 
         logger.info ("##############")
 
-except:
+except Exception as e:
```

Comment:
Smooth!

Reply:
This makes finding errors easier but I still don't love the big try and excepts!